scripts/mbedtls_dev/crypto_knowledge.py

```diff
@@ -34,7 +34,7 @@ def short_expression(original: str, level: int = 0) -> str:
     unambiguous, but ad hoc way.
     """
     short = original
-    short = re.sub(r'\bPSA_(?:ALG|ECC_FAMILY|KEY_[A-Z]+)_', r'', short)
+    short = re.sub(r'\bPSA_(?:ALG|DH_FAMILY|ECC_FAMILY|KEY_[A-Z]+)_', r'', short)
```

Comment:
I don't understand: why backport FFDH support to the python scripts? In 2.28, the PSA API doesn't support FFDH.

Also, the commit "Add DH support for psa test generation and update data files" is doing other things.